atlp-rwanda/commanders-rn-medica
# Feature: Schedule Appointment screen

### What does this pr do ?

This pr provides a screen that helps clients schedule appointment in any way according to their choice of method to be used in an appointment

#### Description of the task to be completed ?
The task will be completed, and the client will be able to use it to meet to a doctor in an easy way, according to his or her choice

#### How should this be manually tested?
 You can use : - npm run test
                         - npm run build

 And you reach my screen, work on  going to my appointment and you see upcoming , complete and canceled appointments. If you want to schedule appointment, you click on button, go to the reason for scheduling the appointment, go to the date and time, and then, by choice, go to cancel your appointment

and you see the icon indicating if you want to choose voice call , messaging or video call

![51_Light_my appointment upcoming-1](https://github.com/atlp-rwanda/commanders-rn-medica/assets/136312774/b6770233-6e27-4973-8ffd-da712df1f63d)

![Screenshot_1716304154](https://github.com/atlp-rwanda/commanders-rn-medica/assets/136312774/ef2944d3-d7fe-4dc0-bc7a-458e2d59a2b1)

![Screenshot_1716304184](https://github.com/atlp-rwanda/commanders-rn-medica/assets/136312774/7cf0f6e7-0b8b-4648-90c2-501231043700)

![Screenshot_1716304191](https://github.com/atlp-rwanda/commanders-rn-medica/assets/136312774/b7582436-2a72-4040-a4e7-79e148950397)
